src/client-tests/README.md

```diff
@@ -0,0 +1,27 @@
+# Client Tests
```

Comment:
I know its usual in this repo to have docs all together, but this seems more straight forward to have information about debugging with the package itself.